src/test/resources/cucumber/features/integrationTests/jira/publish-processing.feature

```diff
@@ -24,6 +24,7 @@ Feature: parse, and then publish processing given SAST XML results, findings sho
     When  publishing results to JIRA
     Then  verify <Total_Of> new issues got created
     And verify <Number_Of> findings in body
```

Comment:
1) what is 'body'?
2) it's better to write that something happens (instead of 'verify')